modules/apps/jaggery/emm/src/emm/units/login/public/js/login-box.js

```diff
@@ -1,10 +1,10 @@
 
-function utf8_to_b64( str ) {
-    return window.btoa(unescape(encodeURIComponent( str )));
+function utf8_to_b64(stringValue) {
```

Comment:
uill.js is a backend js file while this is a front end js file.
for increase re-usability of methods which requires from various units we can put those front js methods into a single unit and include that in every unit which requires those functionalities needed.WDYT?